aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1 +1,361 @@
+
+The following are some high-level guidelines for constructing good messaging:
+
+- Conversational American English
+- Correct grammar and punctuation
+- Active tense
+- Positive, not negative
+- Not robotic sounding
+- Never condescending
+- Establish context
+- Lead with what is most important
+- Inspire trust and confidence
+- Explain using terminology consistent with the UI
+- Avoid charged words like "*error*"
+- Avoid blame casting (on user or on software)
+- State facts without coloring them in opinions
+- Avoid personal pronouns
+- Careful with words like "*Please*"
+- Do not use "*contact your administrator*" (the admin is likely the user) or "*contact HPE*" (HPE might not be the support contact -- it might be a channel partner)
+- Read your message out loud to another person. Is this how you would say it out loud? Does the other person feel this is how two people would converse?
+
+## Capitalization
+
+The general rule is that things that are sentences should follow sentence capitalization rules. And, things that are titles/headings/labels should follow title capitalization rules.
+
+Sentences follow sentence capitalization rules:
+
+- Paragraphs of text
+- Tooltips
+- Notifications
+- Help
+
+Titles follow title capitalization rules:
+
+- Page/dialog titles
+- Section headings
+- Attribute labels
+- Table column headings
+- Legends
+- Tabs/Buttons
+
+Capitalize proper nouns. HPE brand and product names are proper nouns.
+
+Fight the urge to elevate normal nouns to proper noun status. For example, if you are in a "data sources" page and need to say something about data sources, do not capitalize it. Unless it is a true proper noun, leave it uncapitalized when following sentence capitalization rules.
+
+DO NOT USE ALL CAPS IN EITHER TITLES OR SENTENCES. It is harder to read and is equated with yelling.
+
+## Acronyms
+
+Do not use an uncommon acronym in messaging without first defining the acronym the first time it is used. Do not use an acronym like this unless it is going to be used multiple times in the message.
+
+- Incorrect: *The UBC is meeting tomorrow.*
+- Incorrect: *The UBC (university building council) is meeting tomorrow.*
+- Correct: *The UBC (university building council) is meeting tomorrow. If you have UBC-related business, send it to the UBC administrator for addition to the meeting agenda.*
+	
+## Branding
+
+To aid in comprehension, do not use full brand/product names in messaging. Refer to these with shortened names. For example, "*HPE GreenLake LightHouse*" is a mouthful and if needed several times in a sentence, set context and consider using a simpler "*Lighthouse*" the second, third, Nth time.
+
+A best practice for developers is to use parameter substitution in messaging where branded names occur.
+
+- Branding can change over time. E.g. GreenLake Central has recently been rebranded to Cloud Services. If messages had contained `${prodname}`, this would have been an easy fix.
```

Comment:
```suggestion
- Branding can change over time. E.g. GreenLake Central has recently been rebranded to Cloud Services. Implementing messages with `${prodname}` instead of hardcoded messages would make this an easy fix.
```